isb_web/config.py

```diff
@@ -54,5 +54,11 @@ class Settings(BaseSettings):
 
     stac_collection_url_path: str = "stac_collection"
 
+    # The URL to the analytics API
+    analytics_url: str = "UNSET"
+
+    # The domain to record analytics events for, needs to be configured as a site in plausible.io
+    analytics_domain: str = "UNSET"
+
```

Comment:
New config to tell us the URL to post the analytics data, and also which domain we should record data for.

Reply:
@datadavev, I did tell you I think we'd need to define separate hostnames to get this working properly.  I read a bit more on their site, though, and it looks like the domain is just an identifier where we record data.  So, in theory, we could just use this config differently on each iSB instance in henry, and send it along to the API happily.

Reply:
OK, that's not too hard to deal with.